lib/spree/core/controller_helpers/auth.rb

```diff
@@ -43,7 +43,7 @@ def store_location
             end
           end
 
-          disallowed_urls.map!{ |url| url[/\/\w+$/] }
+          disallowed_urls.map!{ |url| url[%r{/\w+$}] }
```

Comment:
This is just replacing the regex `/.../` notation with the `%r{...}` notation, right?

Reply:
yes, correct :+1: 
thanks for the review!